integration_tests/dbt_project.yml

```diff
@@ -13,7 +13,8 @@ dispatch:
 vars: 
   
   apple_search_ads__using_search_terms: True
-  twitter_ads__using_keywords: False
+  twitter_ads__using_keywords: True
```

Comment:
@fivetran-catfritz should this be switched?

Reply:
@fivetran-joemarkiewicz I did it so the docs would generate with the twitter ads keyword reports, but if they shouldn't be I'll put it back.

Reply:
I have changed this back!